### PR TITLE
Added new project 2Channel_LCDBarGraph

### PR DESCRIPTION
**Pull request by Aryan-Prakhar**
Connect the development board with an LCD display to view the Bar Graphs depicting strength from muscles that are connected using 2 different Channels to transfer signals.

### CODE REVIEW

File: 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include <LiquidCrystal_I2C.h>
```
Comment (lorforlinux) [inline 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino:32]: Provide the library author name and link if users cannot decide which library to install.

File: 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+#include <Wire.h> 
+
+// Initialize LCD object with I2C address 0x27, 16 columns and 2 rows
+LiquidCrystal_I2C lcd(0x27, 16, 2);
```
Comment (lorforlinux) [inline 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino:37]: Mention the module used, I2C address can be different between modules/models.

File: 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+};
+
+// Sampling rate in Hz
+#define SAMPLE_RATE 500
+
+// Serial communication baud rate
+#define BAUD_RATE 115200
+
+#define INPUT_PIN1 A0
+#define INPUT_PIN2 A2
```
Comment (lorforlinux) [inline 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino]: Mention that you used Muscle BioAmp Shiled and a Muscle BioAmp Patchy on the Analog port of Shield.

File: README.md [hunk 4]
```diff
@@ -11,6 +11,8 @@ Firmware for Muscle BioAmp hardware from Upside Down Labs
 |6 | [LEDBarGraph](6_LEDBarGraph)| LED bar graph representing EMG Amplitude.|
 |7 | [MuscleStrengthGame](7_MuscleStrengthGame)| Measure your muscle strength. |
 |8 | [EMGScrolling](8_EMGScrolling)| Scrolling through screen using hands. |
+|9 | [2Channel_LCDBarGraph](9_2Channel_LCDBarGraph)| LCD bar graph representing EMG Amplitude using 2 Channel |
```
Comment (lorforlinux) [inline README.md:14]: We can create a diagram to show how Muscle BioAmp Shield and Muscle BioAmp Pathy were connected to use this program.

**Review by lorforlinux — COMMENTED** [review]
@Aryan-Prakhar Please check the comments I have added.

**Comment by Aryan-Prakhar** [comment]
@lorforlinux i have checked the comments and made the updates accordingly. Please review

**Comment by Aryan-Prakhar** [comment]
@lorforlinux I have made all the required correction . Please check and review it.

**Comment by Aryan-Prakhar** [comment]
@lorforlinux I have added a new and more descriptive image for 2 Ch EMG LCD BarGraph

File: 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino (hunk 1 above)
Comment (Aryan-Prakhar) [reply 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino:32]: I have mentioned the library author name as well as their github source link. Please review

File: 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino (hunk 2 above)
Comment (Aryan-Prakhar) [reply 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino:37]: I have mention the exact I2C module name that has been used

File: README.md (hunk 4 above)
Comment (Aryan-Prakhar) [reply README.md:14]: I have added a jpeg image file showing the 2 Channel EMG Signal Acquisition to describe the appropriate connections with pins and ports

**Comment by Aryan-Prakhar** [comment]
@lorforlinux i have resolved all the mentioned changes and add on. Please review the same

File: 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino (hunk 3 above)
Comment (Aryan-Prakhar) [reply 9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino]: I have added comments wherein it is mentioned the pins connections of Muscle Bioamp shield and Muscle Bioamp Patchy

**Comment by lorforlinux** [comment]
@Aryan-Prakhar Thank you so much for your contribution, I am merging this now.

**Pull request merged**